AppCenterCrashes/AppCenterCrashes/Internals/Util/MSACErrorLogFormatter.m

```diff
@@ -312,6 +315,19 @@ + (MSACErrorReport *)errorReportFromLog:(MSACAppleErrorLog *)errorLog {
   NSString *exceptionName = errorLog.exceptionType;
   NSDate *appStartTime = errorLog.appLaunchTimestamp;
   NSDate *appErrorTime = errorLog.timestamp;
+  NSString *codeType = unknownString;
+  NSString *archName = unknownString;
+  if (errorLog.primaryArchitectureId != nil) {
+    codeType = [self convertCodeTypeToString:errorLog.primaryArchitectureId.longValue typeEncode:errorLog.isKnownEncodingType];
```

Comment:
Instead of passing `isKnownEncodingType` param to these functions we can call functions only if `isKnownEncodingType` is `true`. And if it is `false` do something like:
```objc
if (!errorLog.isKnownEncodingType) {
  codeType = @"Unknown";
  archName = @"Unknown";
}
```